general_autoencoder.py

```diff
@@ -492,8 +491,9 @@ def _get_projections_from_processed_data(self, data, ages, project_onto_mean, ro
                     # if we have a closed form for Z_mu, use this for Z. 
                     Z = self.sess.run(self.Z_mu, feed_dict = {self.X:data_i, self.ages:ages_i})
                 else:
```

Comment:
Hm, isn't Z_mu just age * exp(mu + sigma^2/2) for the rate_of_aging model? I'm confused as to whether it's difficult to implement (seems easier than this sampling business)

Reply:
and less hacky too. i will implement and merge.